packages/castmill/lib/castmill/addons/component_info.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Castmill.Addons.ComponentInfo do
+  defstruct name: nil, description: nil, version: nil, icon: nil, mount_point: nil, mount_path: nil
```

Comment:
Should all be nil per default? Do we need validation that some are set?

Reply:
Currently Elixir gives a compiler warning if we forget to set them, no sure how this works, if nil means it is required but if you write a real value then it is optional 🤔